.travis.yml

```diff
@@ -1,42 +0,0 @@
-language: python
-cache: pip
-
-python:
-- '2.7'
-- '3.5'
-- '3.6'
-- '3.7'
-- '3.8'
-- 'nightly'
-
-matrix:
-  allow_failures:
-    - python: 'nightly'
-  fast_finish: true
-
-env:
-  global:
-  - AWS_BUCKET="tweepy"
-  - secure: TPQSFGqdl6khXqQqTZ6euROoAmFRnONAlPXD6npvTIIN+fNfnz8lvZtOEWHo2jRPLoU3FyVUhYvTynj6B2hJinulP+RKOMbQ65HCZVHrsitwl1n1QZB5HegQDOYc5q6VTTYn/r8r5tGy35U0O80y1zycTLqSJiXlkdqsSq564pI=
-
-install:
-  - pip install .[dev,test]
-
-script:
-  - python setup.py nosetests
-
-after_success:
-  - if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then coveralls; fi
```

Comment:
This isn't included in the Test workflow?

Reply:
[coveralls-python 2.1.0](https://github.com/coveralls-clients/coveralls-python/releases/tag/2.1.0) fixes GitHub Actions support, but [2.0.0](https://github.com/coveralls-clients/coveralls-python/releases/tag/2.0.0) dropped support for Python 2.7, so I'll wait [until Python 2 support is dropped for Tweepy](https://github.com/tweepy/tweepy/issues/1253) to add this to the Test workflow.